examples/1_hypercharge_postgres/dozer-config.yaml

```diff
@@ -38,17 +38,31 @@ sources:
       - nasdaq_symbol
       - next_shares
     connection: !Ref stocks
+
+sql: |
+  -- transformation made on top of `stocks` table data
+  SELECT
+    id,
+    AVG(open),
+    AVG(close),
+    low,
+    volume 
+  INTO stocks_data
+  FROM stocks
+  GROUP BY id;
+
 endpoints:
   - name: stocks
     path: /stocks
-    table_name: stocks
+    table_name: stocks_data
     index:
       primary_key:
         - id
-  - id: null
-    name: stocks_meta
+  - name: stocks_meta
     path: /stocks-meta
     table_name: stocks_meta
+    # direct selecting columns at the source level for `stocks_meta`
+    sql: SELECT * from stocks_meta;
```

Comment:
I m guessing this will fail ? There is no SQL property on endpoins anymore.

Reply:
addressed